app/src/main/java/com/example/workoutapp/RetrofitClientInstanceUserandGroup.kt

```diff
@@ -3,11 +3,11 @@ package com.example.workoutapp
 import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 
-object RetrofitClientInstanceUser {
+object RetrofitClientInstanceUserandGroup {
     private var retrofit: Retrofit? = null
 
 
-    private const val BASE_URL = "https://raw.githubusercontent.com/chintavs/Workout-App/main/app/assets/users.json"
+    private const val BASE_URL = "https://raw.githubusercontent.com/chintavs/Workout-App/main/app/assets/"
```

Comment:
Base url change so more than one activity can access the repository

Reply:
Yes... endpoint should be in the DAO, not RetrofitClientInstance.  RetrofitClientInstance should only contain the base URL.